hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCheckAccess.java

```diff
@@ -32,9 +28,14 @@
 import org.apache.hadoop.fs.permission.AclEntryType;
 import org.apache.hadoop.fs.permission.FsAction;
 import org.apache.hadoop.security.AccessControlException;
+
 import org.junit.Assume;
 import org.junit.Test;
 
+import java.io.FileNotFoundException;
```

Comment:
please restore to where they were. java.* should be the first import.

Please: import conflict is one of the key source of false-positive backport problems. Patches MUST NOT move imports around without good reason. This is not one of them.

* If your IDE is set up to rearrange things: disable.
* if your IDE has an "optimize imports" button -don't press it.
* consider reviewing submitted PRs to look for spurious IDE-initiated changes

Reply:
Done